cmd/utils.go

```diff
@@ -1072,6 +1072,13 @@ func getStackLabels(config *RootCommandConfig) (map[string]string, error) {
 		}
 	}
 
+	imageAndDigest := data[0]["RepoDigests"].([]interface{})
+	if len(imageAndDigest) > 0 { //Check that the image has a digest
+		imageAndDigestStr := fmt.Sprintf("%v", imageAndDigest[0])
```

Comment:
since here you are not formatting, youn can use `string(imageAndDigest[0])` within `strings.Split()`